src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/ui-data/lambda-metrics.mdx

```diff
@@ -0,0 +1,385 @@
+---
+title: Available Lamda metrics
+metaDescription: Available Lambda metrics in New Relic.
+freshnessValidatedDate: never
+
+---
+The following table lists the available Lambda metrics for your Lambda functions in New Relic:
+
+<table>
+    <thead>
+        <tr>
+            <th style={{ width: "200px" }}>
+            **Metric type**
```

Comment:
@cbehera-newrelic I think any values in the table heading are automatically converted to bold by the CSS. I'm not sure you need to make them bold with inline formatting.